cfgov/unprocessed/apps/analytics-gtm/webpack-config.js

```diff
@@ -0,0 +1,61 @@
+/* ==========================================================================
+   Settings for webpack JavaScript bundling system.
+   ========================================================================== */
+
+const BROWSER_LIST = require( '../../../../config/browser-list-config' );
```

Comment:
You import statements here.

Reply:
As discussed in chat, webpack config is not run through babel so imports aren't allowed yet.